target/linux/ipq40xx/files-6.1/arch/arm/boot/dts/qcom-ipq4019-whw03.dts

```diff
@@ -0,0 +1,373 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qcom-ipq4019.dtsi"
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/soc/qcom,tcsr.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+	model = "Linksys WHW03 (Velop)";
+	compatible = "linksys,whw03", "qcom,ipq4019";
+
+	aliases {
+		led-boot = &led_blue;
+		led-failsafe = &led_red;
+		led-running = &led_blue;
+		led-upgrade = &led_red;
+	};
+
+	// Default bootargs includes rootfstype=ext4 and needs to be overriden.
+	chosen {
+		bootargs-append = " rootfstype=squashfs clk_ignore_unused";
```

Comment:
Drop `clk_ignore_unused` that is stupid QCA thing they used because they did mark all of the critical clocks as such

Reply:
done!